Sources/Conduit/Networking/Serialization/MultipartFormRequestSerializer.swift

```diff
@@ -72,7 +72,7 @@ public final class MultipartFormRequestSerializer: HTTPRequestSerializer {
             return String(characters)
         }
 
-        return "----------------------------\(randomCharacters.joined())"
+        return randomCharacters.joined()
```

Comment:
This function seems to be used only once, to set the `contentBoundary`.
1. Since it is only used there, could we just put the function into a lazy initialization of that variable?
2. `contentBoundary` itself is used in `inlineContentBoundary` and `finalContentBoundary`, which don't seem like they are ever used. Could we delete those while you are here?

Reply:
1. Good idea and all updated.
2. I might be misunderstanding your comment here because`inlineContentBoundary` & `finalContentBoundary` are used in the function `makeHTTPBody`.